Client/Frontend/Browser/Playlist/PlaylistPopoverView.swift

```diff
@@ -81,14 +80,14 @@ struct PlaylistPopoverView: View {
                 Text(Strings.PlayList.playlistPopoverAddedTitle)
                     .foregroundColor(Color(UIColor.primaryButtonTint))
                     .font(.title2.weight(.medium))
+                    .padding(.bottom, 20.0)
             } else {
                 Text(Strings.PlayList.playlistPopoverAddedTitle)
                     .foregroundColor(Color(UIColor.primaryButtonTint))
                     .font(.headline.weight(.medium))
+                    .padding(.bottom, 20.0)
```

Comment:
Remove `.padding(.bottom, 20)` from all the views, make containing VStack spacing 20

Reply:
Fixed at https://github.com/brave/brave-ios/pull/4508/commits/a51ac89b29bda9b22be0eed68d06ded22e4dfad0